CHANGELOG.en-US.md

```diff
@@ -16,6 +16,33 @@ tag: vVERSION
 
 ---
 
+## 5.19.0
+
+`2024-07-01`
+
+- 🆕 ConfigProvider now supports global configuration for `variant`. [#49535](https://github.com/ant-design/ant-design/pull/49535) [@MadCcc](https://github.com/MadCcc)
+- 🆕 QRCode component now uses `rc-qrcode` instead of `qrcode.react`, adding support for click events and `iconSize` configuration. [#49454](https://github.com/ant-design/ant-design/pull/49454) [#49240](https://github.com/ant-design/ant-design/pull/49240) [@kiner-tang](https://github.com/kiner-tang) [@thinkasany](https://github.com/thinkasany)
```

Comment:
```suggestion
- 🆕 QRCode 
  - uses `rc-qrcode` instead of `qrcode.react`. [#49454](https://github.com/ant-design/ant-design/pull/49454) [@kiner-tang](https://github.com/kiner-tang)
  - adding support for click events and `iconSize` configuration.  [#49240](https://github.com/ant-design/ant-design/pull/49240)  [@thinkasany](https://github.com/thinkasany)
```